### PR TITLE
refactor: unify media queries, remove nested selectors

### PR DESCRIPTION
**Pull request by olehrakwix**
- unify media queries
- flatten all nested selectors
- remove secondaryButton class as it was used for board helper only

### CODE REVIEW

File: src/styles/general.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+//media sizes
+$mobile-width: 480px;
+$tablet-width: 768px;
+$desktop-width: 1024px;
```
Comment (VladyslavG) [inline src/styles/general.scss:4]: did large-desktop disappear?

**Comment by VladyslavG** [comment]
why did you remove CSS nesting ?

File: src/styles/general.scss (hunk 1 above)
Comment (olehrakwix) [reply src/styles/general.scss:4]: yes, we have just mobile, tablet and desktop breakpoints.

```
Unified Media Query Breakpoints:
480, 768, 1024
```

**Comment by olehrakwix** [comment]
@VladyslavG , nesting was removed accidently (I was under impression that we don't support nested css as well as scss selectors in codux, however, it is not true for scss)

but, as we discussed on a meeting, lets keep this style (no nested rules)

**Review by VladyslavG — APPROVED** [review]

**Pull request merged**